### PR TITLE
Release Notes for Magento Commerce 1.14.4.0 and Magento Open Source 1.9.4.0

### PR DESCRIPTION
**Pull request by jfrontain**
## This PR is a:

- [ ] New topic
- [ x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will add release notes for the Magento Open Source 1.9.4.0 and Magento Commerce 1.14.4.0 release.
whatsnew
The [Magento Open Source Release Notes 1.9 and later](https://devdocs.magento.com/guides/m1x/ce19-ee114/ce1.9_release-notes.html) and [Magento Commerce Release Notes 1.14 and later](https://devdocs.magento.com/guides/m1x/ce19-ee114/ee1.14_release-notes.html) provide detailed information about the Magento Open Source 1.9.4.0 and Magento Commerce  1.14.4.0 release.

## Additional information

List all affected URLs 

- ...
- ...

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3205) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3205) before we can accept your contribution.<br/><hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

**Comment by magento-cicd2** [comment]
Can one of the admins verify this patch?

File: guides/m1x/ce19-ee114/ce1.9_release-notes.html [hunk 1]
```diff
@@ -60,6 +61,42 @@ <h2 id="upgrade">Important Upgrade Information</h2>
...
+
+<h2 id="ce19-1940">Magento Open Source 1.9.4.0 Release Notes</h2>
+ <p>This version (or patch SUPEE-10975, which applies to older versions of Magento) provides resolution of multiple critical security issues and functional fixes. These critical security issues include remote code execution (RCE),  cross-site scripting (XSS), and cross-site request forgery (CSRF) issues. This release also provides support for PHP 7.2.</p>
+
+<p>We recommend upgrading your Magento store to this latest version.  See  <a href="https://magento.com/security/patches/supee-10888" target="_blank">Magento Security Center</a> for a comprehensive discussion of these issues.</p>
+
+<p>Note that after updating to this release, third-party modules that depend upon Magento core backup functionality will no longer work. Alternatively, you can use one of these two methods  to enable database backups: </p>
+
+<ul>
+<li>Change <b>false</b> to <b>true</b> in the <b>config</b> > <b>modules</b> > <b>Mage_Backup</b> > <b>active</b>  setting of <code> app/etc/modules/Mage_All.xml</code>. </li>
```
Comment (keharper) [inline guides/m1x/ce19-ee114/ce1.9_release-notes.html]:
```suggestion
<li>In the <b>config</b> > <b>modules</b> > <b>Mage_Backup</b> > <b>active</b>  setting of <code>app/etc/modules/Mage_All.xml</code>, change <b>false</b> to <b>true</b>.</li>
```

File: guides/m1x/ce19-ee114/ce1.9_release-notes.html [hunk 2]
```diff
@@ -60,6 +61,42 @@ <h2 id="upgrade">Important Upgrade Information</h2>
...
+<li><!-- MPERF-10223 -->This release provides support for PHP 7.2. </li>
+<li><!-- MPERF-10217 -->We’ve removed the CC module. As a result, third-party modules that depend upon either the <code>ccsave</code> method or the <code>xmlconnect</code> method will not work as expected.  Third-party themes that implement <code>ccsave</code> will not work as expected, either. </li>
+<li><!-- MPERF-10219 10218 -->The Magento logo has been updated throughout the code base. </li>	
+<li><!-- MPERF-8829 -->The <b>Continue</b> button now works as expected on the Payments step of checkout  when paying with the PayPal payment method. </li>	
+<li><!-- MPERF-8826 -->Google Tag Manager now logs sales information in Google Analytics as expected. </li>	
+<li><!-- MPERF-8382 -->The product export CSV file now contains columns for super attributes. </li>	
+<li><!-- MPERF-7181 -->Magento no longer throws an error when a customer accesses their shopping cart after items in their cart have been removed due to a timeout. Previously, Magento displayed this error, `Notice: Undefined variable: freePackageValue in /var/www/dev/htdocs/app/code/core/Mage/Shipping/Model/Carrier/Tablerate.php on line 130`.  </li>
```
Comment (keharper) [inline guides/m1x/ce19-ee114/ce1.9_release-notes.html]: I don't think the backticks will render correctly in a browser. Replace with `<code>` and `</code>`

File: guides/m1x/ce19-ee114/ce1.9_release-notes.html [hunk 3]
```diff
@@ -60,6 +61,42 @@ <h2 id="upgrade">Important Upgrade Information</h2>
...
+<li><!-- MPERF-8826 -->Google Tag Manager now logs sales information in Google Analytics as expected. </li>	
+<li><!-- MPERF-8382 -->The product export CSV file now contains columns for super attributes. </li>	
+<li><!-- MPERF-7181 -->Magento no longer throws an error when a customer accesses their shopping cart after items in their cart have been removed due to a timeout. Previously, Magento displayed this error, `Notice: Undefined variable: freePackageValue in /var/www/dev/htdocs/app/code/core/Mage/Shipping/Model/Carrier/Tablerate.php on line 130`.  </li>	
+<li><!-- MPERF-7067 -->Clicking on a configurable product’s swatch on the  product list page now updates product price as expected.  </li>	
+<li><!-- MPERF-6629 -->Customers can now successfully add a grouped product to their shopping cart when category permissions are enabled. Previously, Magento did not add the product to the cart, but instead displayed a descriptive error message.  </li>	
+<li><!-- MPERF-6566 -->Magento no longer displays incorrect prices on the storefront after a failure of the enterprise refresh index.  </li>
```
Comment (keharper) [inline guides/m1x/ce19-ee114/ce1.9_release-notes.html]: Verify whether "enterprise refresh index" belongs in the CE release notes.

Comment (jfrontain) [reply guides/m1x/ce19-ee114/ce1.9_release-notes.html]: Verified with PM that this note belongs in both CE and EE.

**Pull request merged**